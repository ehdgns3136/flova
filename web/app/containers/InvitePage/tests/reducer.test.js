
import { fromJS } from 'immutable';
import invitePageReducer from '../reducer';

describe('invitePageReducer', () => {
  it('returns the initial state', () => {
    expect(invitePageReducer(undefined, {})).toMatchSnapshot();
  });
});
