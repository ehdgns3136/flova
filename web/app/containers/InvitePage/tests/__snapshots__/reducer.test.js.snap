exports[`invitePageReducer returns the initial state 1`] = `
Object {
  "inviteKey": null,
  "invitedName": null,
  "isValidating": false,
  "validateError": null,
}
`;
